lib/scrape.js

```diff
@@ -340,6 +364,32 @@ async function fetchDates() {
     (a, b) => b.completed_task_instance_count - a.completed_task_instance_count
   )
 
+  Promise.all(
```

Comment:
Probably looks neater if you put it inside variable first and then use `Promise.all()`.

Also, async function please, instead of `then`.